rotkehlchen/tests/api/test_evm_transactions.py

```diff
@@ -89,3 +125,111 @@ def test_query_transactions(rotkehlchen_api_server: 'APIServer'):
         for entry in result['entries']:
             assert entry['entry']['timestamp'] <= last_ts
             last_ts = entry['entry']['timestamp']
+
+
+@pytest.mark.parametrize('ethereum_accounts', [[ADDY]])
+@pytest.mark.parametrize('is_async_query', [True, False])
```

Comment:
Don't do this as argument here since it will trigger the test to run twice and is not needed. I meant to do something like in https://github.com/rotki/rotki/blob/a1a43b69966ceb153a53dbda3446e3ff9fc42090/rotkehlchen/tests/api/test_exchanges.py#L492

this is enough for us to notice if there is any issue